src/python/grpcio/grpc/_channel.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, due, initial_metadata, trailing_metadata, code, details):
         self.trailing_metadata = trailing_metadata
         self.code = code
         self.details = details
+        self.debug_error_string = ""
```

Comment:
Avoid assigning a value to a field until after determining that the value is the right value to assign to the field. The empty string is a legitimate value for the details of an RPC. Why not use something like `None` or `_MY_CONSTANT_REPRESENTING_NOT_YET_DETERMINED` here? (Probably `None`, but I'm open to arguments about other values...)

Reply:
Done